storage/src/vespa/storage/persistence/filestorage/filestormanager.cpp

```diff
@@ -765,13 +765,13 @@ FileStorManager::sendReply(const std::shared_ptr<api::StorageReply>& reply)
     // Currently we need to dispatch due to replies sent by remapQueue
     // function in handlerimpl, as filestorthread keeps bucket db lock
     // while running this function
-    dispatchUp(reply);
+    sendUp(reply);
```

Comment:
The comment above this call implies that not dispatching could cause a deadlock. Has the threading/locking been changed sufficiently that this no longer is a problem? If so, the comment should be removed. If not, some more pondering might be needed.

Reply:
You are right. Reverting and just doing GC of members.